packages/app-store/wipemycalother/components/index.ts

```diff
@@ -1,2 +1 @@
-export { default as InstallAppButton } from "./InstallAppButton";
```

Comment:
@sean-brydon There still was InstallAppButton file for stripe app and thus isOmniInstall was being treated as false for it always. This was the reason, even with wrong logic stripe app was working correctly 🤦‍♂️

Reply:
lol I guess that makes sense - thanks for solving the head scratcher